_posts/2021-04-23-release-2.22.0.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Release v2.22.0
+tags: ["v2.22.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 23/04 2021 the new version 2.22.0 is out
+<!--more-->
+
+## Standardization Improvements
+
+- [#1753]({{ site.github.issues_url }}/1753) Add an option to ignore trailing and leading white spaces on CSV read
+
+## Conformance Improvements
+
+- [#1662]({{ site.github.issues_url }}/1662) `MappingConformanceRule` now allows having multiple output columns from mapping tables. The implementations are done for Broadcast and GroupExplode
+
+## Menas Improvements
+
+- [#1702]({{ site.github.issues_url }}/1702) Dataset properties essentiality level Mandatory enhanced with parameter allowRun. (If set to true SparkJobs task will still execute even if the property is not defined. Menas UI configuration validation will fail though.)
+
+## Helper Scripts Improvements
+
+- [#1759]({{ site.github.issues_url }}/1759) Have a more strict DRA setup in bash helper scripts. Differentiate between spark job run-time parameters for DRA and non-DRA setup and if DRA set up is enabled or defaulted, ignore the non-DRA number of executors, memory and cores.
+
+## Project Testing
+
+- [#1625]({{ site.github.issues_url }}/1625) Now it is possible to populate test data into HDFS and MongoDB by bash scripts. Data are part of the codebase. It is possible to run tests from codebase with test JSONs using Hermes 0.3.1+
```

Comment:
```suggestion
- [#1625]({{ site.github.issues_url }}/1625) Now it is possible to populate test data into HDFS and MongoDB by bash scripts, since the data is part of the codebase. It is possible to run tests from codebase with test JSONs using Hermes 0.3.1+
```

Reply:
Changed the suggestion to _data are_ - the customary way in computer science.